### ISSUE TITLE
Email open bug

### ISSUE THREAD

**Issue by Augentia** [mautic/mautic#2407]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Bug on Email open
| Feature request? | 
| Enhancement? | 

## Description:
Bug spotted in Mautic. The test case is this. Send an email to a contact. After email is sent - open the Contact details --> go to History --> Expand the 'Email sent' and click on the email sent link --> it opens the email copy in a different window --> now come back to the contact & refresh --> it shows the contact opened the email (when the Mautic user clicks the email link - Mautic treats this as an email opened by contact). Please correct this bug.

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 
| PHP version | 

### Steps to reproduce:
1. Send email to a contact
2. Open contact
3. Go to History
4. Expand the 'Email sent' and click on the email sent link
5. It opens the email copy in a different window
6. Come back to contact
7. Mautic shows the email is opened by contact (though it was only viewed by Mautic user)
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by escopecz** [comment]
I remember this was fixed already. You forgot to fill in what Mautic version and PHP version you use.

**Comment by Augentia** [comment]
@escopecz I am using mautic.com ...tested it again today and not able to replicate the bug. Thanks for addressing it.

**Comment by Augentia** [comment]
This bug resurfaced again. Today I was checking on an email that was sent to a contact in our Mautic instance. The contact ID is 9994 and we sent a custom email that was sent on Sep 14 (email subject: SFDC Marketing Cloud Delivery partnerships). Mautic was showing the email was not read yet. I clicked on the email link. It opened in a new window. After a moment I closed the new window, and came back to mautic instance, and it showed the email read on Sep 19 (the same time as I opened the email to see its content).
